pkg/client/cmd/team.go

```diff
@@ -196,3 +208,34 @@ func teamRemoveUser(cmd *cobra.Command, args []string) {
 
 	fmt.Printf("User %s has been removed from the team %s\n", color.CyanString(user), color.CyanString(team))
 }
+
+func teamRename(cmd *cobra.Command, args []string) {
+	oldTeam, err := cmd.Flags().GetString("old")
+	if err != nil {
+		client.PrintErrorAndExit("Invalid old team name parameter")
+	}
+
+	newTeam, err := cmd.Flags().GetString("new")
+	if err != nil {
+		client.PrintErrorAndExit("Invalid new team name parameter")
+	}
+
+	if oldTeam == "" || newTeam == "" {
+		cmd.Usage()
+		return
+	}
+
+	conn, err := connection.New(cfgFile, cfgCluster)
+	if err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
+	}
+	defer conn.Close()
+
+	cli := teampb.NewTeamClient(conn)
+	req := &teampb.RenameRequest{OldName: oldTeam, NewName: newTeam}
+	if _, err := cli.Rename(context.Background(), req); err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
```

Comment:
I guess this should be `client.PrintErrorAndExit(client.GetErrorMsg(err))`, right?

Reply:
🤦‍♂️ you're right!